app/views/dump.py

```diff
@@ -0,0 +1,33 @@
+from flask_login import current_user
+from flask_login import login_required
+from flask import Blueprint, jsonify
+
+
+from app.authentication.roles import role_required
+from app.globals import get_answer_store, get_metadata
+from app.submitter.converter import convert_answers
+from app.utilities.schema import get_schema
+from app.questionnaire.path_finder import PathFinder
+
+
+dump_blueprint = Blueprint('dump', __name__)
+
+
+@dump_blueprint.route('/dump/answers', methods=['GET'])
+@login_required
+@role_required('dumper')
```

Comment:
Not easy to do; flush doesn't use the normal login route or user/metadata; we should consider whether to update that later I think